tools/c7n_azure/c7n_azure/resources/network_security_group.py

```diff
@@ -210,8 +210,8 @@ def process(self, network_security_groups):
             resource_group = nsg['resourceGroup']
             for rule in nsg['properties']['securityRules']:
                 self.manager.log.info(
-                    'Updating access to \'%s\' for security rule '
-                    '\'%s\' in resource group \'%s\''.format(
+                    'Updating access to \'{}\' for security rule '
```

Comment:
two more minor comments, instead of escaping quotes. its more common to use double quotes on the string boundaries and use single quotes in the string without escaping.

also its more common to just %s and not format the string, ie.
```
log.info("message '%s'", my_message)
```
the logging framework will lazy construct the string if the configured log level is sufficient for the message to be logged.

Reply:
@kapilt thanks for the suggestion. just pushed a commit formatted the way you discussed.